droplet/bin/restart_proxy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+TORRUST_INDEX_CONFIG_PATH=$(cat ./storage/index/etc/index.toml) \
```

Comment:
This the _PATH variable, but we place the config directly on it here...

Reply:
Good catch @da2ce7! In fact, it's working because that's the old name. New names are:

Tracker:

- `TORRUST_TRACKER_CONFIG`
- `TORRUST_TRACKER_PATH_CONFIG`

Index:

- `TORRUST_INDEX_CONFIG`
- `TORRUST_INDEX_PATH_CONFIG`

I replaced `TORRUST_INDEX_CONFIG_PATH` with `TORRUST_INDEX_PATH_CONFIG` to follow your latest changes.

In this case those variables are not being used in the `compose.yaml` file. I'm using the default location for the config files:

- `TORRUST_TRACKER_PATH_CONFIG=/etc/torrust/tracker/tracker.toml`
- `TORRUST_INDEX_PATH_CONFIG=/etc/torrust/index/index.toml`

Config env vars are set at runtime: 

- Tracker: https://github.com/torrust/torrust-tracker/blob/develop/share/container/entry_script_sh#L54
- Index: https://github.com/torrust/torrust-index/blob/develop/share/container/entry_script_sh#L54

We have to remove `TORRUST_INDEX_CONFIG_PATH` and `TORRUST_TRACKER_CONFIG_PATH` from all bash scripts.